dev/TwoPaneView/InteractionTests/TwoPaneViewTests.cs

```diff
@@ -146,7 +146,7 @@ public void RegionTest()
             {
                 Log.Comment("Verify horizontal split regions");
                 SetComboBox("SimulateComboBox", "LeftRight");
-                
```

Comment:
nit: Stray newline

Reply:
Can you give more info?

No new lines have been added/deleted. All of these lines had many space characters and were not consistent with the rest of the code and I suppose we don't want to keep them this way.